### DISCUSSION TITLE
SelfSupervisedLoss with a miner?

### DISCUSSION THREAD

**Discussion by ManuelZ** [KevinMusgrave/pytorch-metric-learning#713] [Q&A]
Is it possible to use `SelfSupervisedLoss` with a miner? I'm wrapping `CircleLoss`.

**Comment by KevinMusgrave** [comment]
Unfortunately no. But it's a pretty simple wrapper, so the best workaround for now is to copy/paste the label-creating logic and not use the wrapper.

https://github.com/KevinMusgrave/pytorch-metric-learning/blob/28d5acce05f61b0d0c15acbe423a7ba387efcb4d/src/pytorch_metric_learning/losses/self_supervised_loss.py#L57-L67
